Bottom: a587bd5583d1a6e7d66435816e66f5ca7402ed21
Top:    41807b5e8cb9907bb39049625776af868ce482d6
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-05-07 21:51:42 -0400

Refresh of efika-audio

---

diff --git a/sound/soc/fsl/efika-audio-fabric.c b/sound/soc/fsl/efika-audio-fabric.c
index c3e9da0..4726064 100644
--- a/sound/soc/fsl/efika-audio-fabric.c
+++ b/sound/soc/fsl/efika-audio-fabric.c
@@ -24,7 +24,6 @@
 #include <sound/initval.h>
 #include <sound/soc.h>
 #include <sound/soc-of-simple.h>
-#include <sound/soc-dapm.h>
 
 #include "mpc5200_dma.h"
 #include "mpc5200_psc_ac97.h"
@@ -39,6 +38,8 @@
 
 #define DRV_NAME "efika-psc-ac97"
 
+#if 0
+
 static int efika_jack_func;
 static int efika_spk_func;
 
@@ -47,7 +48,7 @@ static void efika_ext_control(struct snd_soc_codec *codec)
 	/* set up jack connection */
 	switch (efika_jack_func) {
 	case EFIKA_HP:
-		snd_soc_dapm_disable_pin(codec, "Headset Jack");
+		//snd_soc_dapm_disable_pin(codec, "Headset Jack");
 		snd_soc_dapm_disable_pin(codec, "Mic (Internal)");
 		snd_soc_dapm_enable_pin(codec, "Headphone Jack");
 		break;
@@ -184,6 +185,13 @@ static int efika_init(struct snd_soc_codec *codec)
 	return 0;
 }
 
+#else
+static int efika_init(struct snd_soc_codec *codec)
+{
+	return 0;
+}
+#endif
+
 /*
  * This is an example codec initialization for a stac9766 connected to a
  * mpc5200. It is missing logic to detect hp/mic insertions and logic
